[2024-05-06T07:31:00.441+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-06T07:31:00.537+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tweets_dag_v4.execute_python_script manual__2024-05-06T07:26:44.121518+00:00 [queued]>
[2024-05-06T07:31:00.561+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tweets_dag_v4.execute_python_script manual__2024-05-06T07:26:44.121518+00:00 [queued]>
[2024-05-06T07:31:00.562+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-06T07:31:00.607+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): execute_python_script> on 2024-05-06 07:26:44.121518+00:00
[2024-05-06T07:31:00.635+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1161) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-06T07:31:00.632+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tweets_dag_v4', 'execute_python_script', 'manual__2024-05-06T07:26:44.121518+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/twitter_data_dag.py', '--cfg-path', '/tmp/tmp1c9kpai3']
[2024-05-06T07:31:00.643+0000] {standard_task_runner.py:91} INFO - Job 130: Subtask execute_python_script
[2024-05-06T07:31:00.641+0000] {standard_task_runner.py:63} INFO - Started process 1318 to run task
[2024-05-06T07:31:00.835+0000] {task_command.py:426} INFO - Running <TaskInstance: tweets_dag_v4.execute_python_script manual__2024-05-06T07:26:44.121518+00:00 [running]> on host 5efa3aaee11b
[2024-05-06T07:31:01.107+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stan' AIRFLOW_CTX_DAG_ID='tweets_dag_v4' AIRFLOW_CTX_TASK_ID='execute_python_script' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T07:26:44.121518+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T07:26:44.121518+00:00'
[2024-05-06T07:31:01.116+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-06T07:37:20.647+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2024-05-06T07:37:20.729+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-06T07:37:20.741+0000] {process_utils.py:132} INFO - Sending 15 to group 1318. PIDs of all processes in the group: [1318]
[2024-05-06T07:37:20.744+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1318
[2024-05-06T07:37:20.752+0000] {taskinstance.py:2607} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-05-06T07:37:20.764+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-06T07:37:20.799+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/scripts/extract_transform.py", line 50, in roberta_classifier
    output = model(**encoded_text)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/transformers/models/roberta/modeling_roberta.py", line 1191, in forward
    outputs = self.roberta(
              ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/transformers/models/roberta/modeling_roberta.py", line 821, in forward
    embedding_output = self.embeddings(
                       ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/transformers/models/roberta/modeling_roberta.py", line 125, in forward
    embeddings = self.LayerNorm(embeddings)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/modules/normalization.py", line 201, in forward
    return F.layer_norm(
           ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/torch/nn/functional.py", line 2573, in layer_norm
    return torch.layer_norm(input, normalized_shape, weight, bias, eps, torch.backends.cudnn.enabled)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 2609, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-05-06T07:37:20.815+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=tweets_dag_v4, task_id=execute_python_script, execution_date=20240506T072644, start_date=20240506T073100, end_date=20240506T073720
[2024-05-06T07:37:20.958+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1318, status='terminated', exitcode=2, started='07:31:00') (1318) terminated with exit code 2
